### ISSUE TITLE
SPDK 20.04.1 - json config bdev_raid_create expects raid_level to be a string, not a number as shown in the docs

### ISSUE THREAD

**Issue by nufosmatic** [spdk/spdk#1518]
In the JSON-RPC methods doc, the "raid_level" example is a number. The decoder decode_raid_level() is expecting a string.

## Expected Behavior

This syntax in a json configuration file should work as documented:

```
{
"method": "bdev_raid_create",
  "params": {
    "name": "Raid0",
    "raid_level": 0,
    "base_bdevs": [
      "Malloc0",
      "Malloc1",
      "Malloc2",
      "Malloc3"
    ],
    "strip_size": 4096
}

```

## Current Behavior

This syntax will actually make it through the decoder:

```
{
"method": "bdev_raid_create",
  "params": {
    "name": "Raid0",
    "raid_level": "0",
    "base_bdevs": [
      "Malloc0",
      "Malloc1",
      "Malloc2",
      "Malloc3"
    ],
    "strip_size": 4096
}

```

## Possible Solution

There appears to be some parameter disagreement between the procedure call, the RPC, and the JSON output from the RPC report on the various API entries. This leads to confusion on the part of the noobies trying to read between the lines where there isn't a good example and would put off the less less tenacious ones.

## Steps to Reproduce

Now that I've gotten past this error/bug/issue/feature, I have a new error/bug/issue/feature I have to get past before I know any of this actually works.

## Context (Environment including OS version, SPDK version, etc.)

Centos 7.5 SPDK 20.04.1

**Comment by jimharris** [comment]
Thanks for the report.  https://review.spdk.io/gerrit/c/spdk/spdk/+/3581 fixes the documentation bug.  This will likely be merged for the 20.07 release, but probably not back ported to the 20.04.x branch.  The patch should apply cleanly to 20.04.1 however.

**Comment by tomzawadzki** [comment]
Patch https://review.spdk.io/gerrit/c/spdk/spdk/+/3581 fixing this issue was merged.
